commands/publish_dafsa.py

```diff
@@ -40,13 +41,12 @@ def download_resources(directory, *urls):
                 f.write(chunk)
 
 
-def get_stored_hash(client):
+def get_stored_hash(client, bucket=None):
     record = {}
     try:
-        record = client.get_record(id=RECORD_ID)
-    except KintoException as e:
-        if e.response is None or e.response.status_code == 404:
-            raise
+        record = client.get_record(id=RECORD_ID, bucket=bucket)
+    except KintoException:
+        record = {}
```

Comment:
You could just put `pass` here. Do we really intend to squash all KintoExceptions here?